src/main/java/com/laserfiche/repository/api/clients/impl/EntriesClientImpl.java

```diff
@@ -1,18 +1,21 @@
 package com.laserfiche.repository.api.clients.impl;
 
-import com.laserfiche.repository.api.clients.EntriesClient;
-import com.laserfiche.repository.api.clients.impl.model.*;
-import kong.unirest.Header;
-import kong.unirest.UnirestInstance;
+import java.util.*;
```

Comment:
Not all imports are needed here. If you are using IntelliJ, you can just hit ctrl+shift+alt+l and also check "optimize imports".